language/move-prover/tests/sources/functional/bitwise_features.move

```diff
@@ -75,13 +72,21 @@ module TestFeatures {
         while (i < n) {
             set(features, *vector::borrow(&disable, i), false);
             spec {
-                assert ((int2bv(((1 as u8) << ((disable[i] % (8 as u64)) as u64) as u8)) as u8)
-                    & features[disable[i] / 8] as u8) == (0 as u8);
+                assert spec_compute_feature_flag(features, disable[i]) == (0 as u8);
             };
             i = i + 1
         };
     }
 
+    spec fun spec_compute_feature_flag(features: vector<u8>, feature: u64): u8 {
```

Comment:
No need to specify which parameter is to be encoded in `bv` (like using `pragma bv=...`)?

Reply:
Unfortunately spec functions does not support pragma so the info needs to be inferred automatically.